examples/multisig_plain/lib.rs

```diff
@@ -0,0 +1,837 @@
+    }
+
+    /// Emitted when a transaction was canceled.
+    #[ink(event)]
+    struct Cancelation {
+        /// The transaction that was canceled.
+        #[ink(topic)]
+        transaction: TransactionId,
+    }
+
+    /// Emitted when a transaction was executed.
+    #[ink(event)]
+    struct Execution {
+        /// The transaction that was executed.
+        #[ink(topic)]
+        transaction: TransactionId,
+        /// Indicates whether the transaction executed successfully.
+        #[ink(topic)]
+        result: Result<(), ()>,
+    }
+
+    /// Emitted when an owner is added to the wallet.
+    #[ink(event)]
+    struct OwnerAddition {
+        /// The owner that was added.
+        #[ink(topic)]
+        owner: AccountId,
+    }
+
+    /// Emitted when an owner is removed from the wallet.
+    #[ink(event)]
+    struct OwnerRemoval {
+        /// The owner that was removed.
+        #[ink(topic)]
+        owner: AccountId,
+    }
+
+    /// Emitted when the requirement changed.
+    #[ink(event)]
+    struct RequirementChange {
+        /// The new requirement value.
+        new_requirement: u32,
+    }
+
+    impl MultisigPlain {
+        /// The only constructor of the contract.
+        /// A list of owners must be supplied and a number of how many of them must
+        /// confirm a transaction. Duplicate owners are silently dropped.
+        #[ink(constructor)]
+        fn new(&mut self, owners: Vec<AccountId>, requirement: u32) {
+            for owner in &owners {
+                self.is_owner.insert(*owner, ());
+                self.owners.push(*owner);
+            }
+            self.ensure_requirement_is_valid(self.owners.len(), requirement);
+            assert!(self.is_owner.len() == self.owners.len());
+            self.requirement.set(requirement);
+        }
+
+        /// Add a new owner to the contract.
+        /// Panics if the owner already exists.
+        /// Only callable by the wallet itself.
+        #[ink(message)]
+        fn add_owner(&mut self, new_owner: AccountId) {
+            self.ensure_from_wallet();
+            self.ensure_no_owner(&new_owner);
+            self.ensure_requirement_is_valid(self.owners.len() + 1, *self.requirement);
+            self.is_owner.insert(new_owner, ());
+            self.owners.push(new_owner);
+            self.env().emit_event(OwnerAddition { owner: new_owner });
+        }
+
+        /// Remove an owner from the contract.
+        /// Only callable by the wallet itself. If by doing this the amount of owners
+        /// would be smaller than the requirement it is adjusted to be exactly the
+        /// number of owners. Panics if `owner` is no owner of the wallet.
+        #[ink(message)]
+        fn remove_owner(&mut self, owner: AccountId) {
+            self.ensure_from_wallet();
+            self.ensure_owner(&owner);
+            let len = self.owners.len() - 1;
+            let requirement = u32::min(len, *self.requirement.get());
```

Comment:
```suggestion
            let requirement = u32::min(len, *self.requirement);
```

Reply:
Maybe I am wrong but it should generally work without the `.get()`

Reply:
You are right.